.circleci/config.yml

```diff
@@ -1,5 +1,7 @@
-
+---
 version: 2.1
+orbs:
+  samvera: samvera/circleci-orb@0
 executors:
```

Comment:
Can this executers section be removed since the job uses the executor from the orb?